hoodie-common/pom.xml

```diff
@@ -70,6 +70,18 @@
   </build>
 
   <dependencies>
+    <dependency>
+      <groupId>com.beust</groupId>
+      <artifactId>jcommander</artifactId>
+      <scope>test</scope>
+      <version>1.48</version>
+    </dependency>
+
+    <dependency>
```

Comment:
same thing.. whats footprint of rocksdb jars.. I am thinking this should be incurred only by hoodie-client ... is it possible to pull out these out of hoodie-common